cli/src/main/kotlin/commands/AdvisorCommand.kt

```diff
@@ -105,8 +105,8 @@ class AdvisorCommand : CliktCommand(name = "advise", help = "Check dependencies
         }
 
         val distinctProviders = providerFactories.distinct()
-
-        println("Using provider '${distinctProviders.first().providerName}'.")
+        println("The following vulnerability providers are activated:")
```

Comment:
I would rather combine this change with a previous commit, but do not have any strong feelings here.

Reply:
Actually, this should be squashed to the *first* commit in this PR, not to the previous commit.